proto/prysm/v1alpha1/attestation/aggregation/testing/bitlistutils.go

```diff
@@ -93,7 +93,7 @@ func MakeAttestationsFromBitlists(bl []bitfield.Bitlist) []*ethpb.Attestation {
 }
 
 // MakeSyncContributionsFromBitVector creates list of sync contributions from list of bitvector.
-func MakeSyncContributionsFromBitVector(bl []bitfield.Bitvector128) []*ethpb.SyncCommitteeContribution {
```

Comment:
This had to change because I included `proposer_utils_bench_test.go` in the test bazel rule in the core package, which was not present in the v1alpha1 package. Including the file led to incompatibility with `minimal` config.